### PR TITLE
Added support for get time API.

### PR DESCRIPTION
**Pull request by sankarsubramanian**
Ready for Review

### CODE REVIEW

**Comment by kristinpeterson** [comment]
@omab can you please review this when you have a moment?

File: examples/get_time.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Instantiate a client
+client = M2XClient(key=API_KEY)
+
+# Instantiate time for the client
+time = Time(client)
+
+# Getting the Time in various formats
+print "Server time in ISO8601 format: %s" %time.get_time_in_iso8601()
+print "Server time in seconds: %s" %time.get_time_in_seconds()
+print "Server time in milliseconds: %s" %time.get_time_in_millis()
```
Comment (omab) [inline examples/get_time.py]: This code should be compatible with Python 3 too like it's stated in the README.

**Comment by sankarsubramanian** [comment]
Requested Changes are Completed. Ready for Review

**Comment by omab** [comment]
@sankarsubramanian I've approved the PR.

About the `print` statements, while this is correct:

```python
print("foo" + "bar")
```

the preferred way is to use the [format](https://docs.python.org/2/library/stdtypes.html#str.format) method:

```python
print("foo {0}".format("bar"))
```

**Pull request merged**